src/Hypsography/Hypsography.jl

```diff
@@ -156,7 +160,46 @@ function ExtrudedFiniteDifferenceSpace(
     )
 end
 
-
+"""
+  diffuse_surface_elevation!(f; κ,iter,dt)
```

Comment:
Needs 4 spaces for indentation
```suggestion
    diffuse_surface_elevation!(f; κ,iter,dt)
    
```

Reply:
Also docstring needs to be added to docs, and make sure that the markdown renders as you intend it.

Reply:
Noted, thanks